smartcart/src/components/SearchBarIcon.js

```diff
@@ -17,21 +32,57 @@ const SearchBarIcon = () => {
         padding: '10px',
         borderRadius: '30px',
         width: 780,
-        height: 180,
         flexDirection: 'row',
-        display : 'flex',
+        display: 'flex',
         justifyContent: 'center',
         alignItems: 'center',
     };
 
+    // List container styles
+    const listContainerStyle = {
+        backgroundColor: '#ffffff',
+        boxShadow: '0px 8px 16px rgba(0, 0, 0, 0.1)',
+        borderRadius: '4px',
+        padding: '10px',
+        marginTop: '10px',
+    };
+
+    const listItemStyle = {
+        borderBottom: '1px solid #e0e0e0',
+        padding: '8px',
+        color: 'black',
+    };
+
     return (
-        <div style = {outerContainerStyle}>
-            <div style = {innerContainerStyle}>
-                <input type="text" id="search" name="search" style={{width: 700, height: 100, fontSize: 50}}/>
-                <img src="https://uxwing.com/wp-content/themes/uxwing/download/user-interface/search-icon.png" style={{width: 100, height: 'auto', padding: '10px', float: 'right'}}/>
+        <div style={outerContainerStyle}>
+            <div style={innerContainerStyle}>
+                <input
+                    type="text"
+                    id="search"
+                    name="search"
+                    style={{ width: 700, height: 100, fontSize: 50 }}
+                    value={searchTerm}
+                    onChange={(e) => setSearchTerm(e.target.value)}
+                />
+                <img
+                    src="https://uxwing.com/wp-content/themes/uxwing/download/user-interface/search-icon.png"
+                    style={{ width: 100, height: 'auto', padding: '10px', float: 'right' }}
+                />
+            </div>
+
+            {/* List of Related Items */}
+            <div style={listContainerStyle}>
+                <ul style={{ listStyle: 'none', padding: 0, margin: 0 }}>
```

Comment:
add '{searchTerm &&' to wrap the list of related items div (before line 74) to conditionally display it based on input.